.github/pull_request_template.md

```diff
@@ -5,3 +5,5 @@
 - [ ] Did you bump the version in `package.json` so that releases to npm are
       made without version conflict?
   - [ ] Did you also run `npm install` to update the version number in the `package.lock`?
+- [ ] Make sure to cut a release so that npm can pick the new version
+  - [ ] Did you also pull down discovery-ui to update the version number?
```

Comment:
How about

```
Reminder:
- If you'd like to see this code deployed after merge, don't forget to cut a release in this repo, then update the package versions in [discovery-ui](https://github.com/UrbanOS-public/discovery_ui)
```

Discovery UI has its own documented steps, so all that needs to happen here in react-disc-ui is cutting the release.

Making changes here also don't necessarily require changes discovery-ui in some cases, so I'd like to see the note as a "reminder" and not a merge requirement in the checklist. Especially since you can't alter disc-ui to reflect changes made here, until the PR is approved and merged in the first place.

Thoughts?

Reply:
When I was going through, I found 0 documentation about steps for Discovery UI, which is y I set up the reminder here. and I'll update the text